.github/workflows/publish-cloud.yml

```diff
@@ -0,0 +1,146 @@
+          CHANNEL: "cloud"
+      - name: Push version
+        uses : maork-elementor/elementor-editor-github-actions/push-new-version@master
+        with:
+          MAINTAIN_USERNAME: ${{ secrets.MAINTAIN_USERNAME }}
+          MAINTAIN_EMAIL: ${{ secrets.MAINTAIN_EMAIL }}
+          COMMIT_MESSAGE: "Bump ${{ inputs.PACKAGE_VERSION || env.PACKAGE_VERSION }}"
+  publish-to-cloud:
+    needs: bump-version
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.MAINTAIN_TOKEN }}
+          repository: ${{ github.event.client_payload.repository}}
+      - name: Install Node.js 14.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+      - name: Get next release branch
+        uses: maork-elementor/elementor-editor-github-actions/get-next-release-branch@master
+        with:
+          INPUT_VERSION: ${{ github.event.inputs.version }}
+      - name: Checkout next release branch
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.MAINTAIN_TOKEN }}
+          ref: ${{ env.RELEASE_BRANCH }}
+          repository: ${{ github.event.client_payload.repository}}
+      - name: Install Dependencies
+        run: |
+          PACKAGE_VERSION=$(node -p "require('./package.json').last_cloud_version")
+          echo "PACKAGE_VERSION=${PACKAGE_VERSION}" >> $GITHUB_ENV
+          npm ci
+      - name: Build plugin
+        uses: maork-elementor/elementor-editor-github-actions/build-plugin@master
+        with:
+          PACKAGE_VERSION: ${{ env.PACKAGE_VERSION }}
+          REPO_NAME: ${{ github.event.client_payload.repository }}
+      - name: Upload zip file to GitHub actions artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ env.PLUGIN_ZIP_FILENAME }}
+          path: ${{ env.PLUGIN_ZIP_PATH }}
+          if-no-files-found: error
+      - name: Create Git tag
+        uses : maork-elementor/elementor-editor-github-actions/create-git-tag@master
+        with:
+          MAINTAIN_USERNAME: ${{ secrets.MAINTAIN_USERNAME }}
+          MAINTAIN_EMAIL: ${{ secrets.MAINTAIN_EMAIL }}
+          PACKAGE_VERSION: ${{ env.PACKAGE_VERSION }}
+      - name: Generate changelog
+        uses : maork-elementor/elementor-editor-github-actions/generate-changelog@master
+        with:
+          TOKEN: ${{ secrets.MAINTAIN_TOKEN }}
+          REPOSITORY: ${{ github.event.client_payload.repository}}
+          HEAD_BRANCH_NAME: v${{ env.PACKAGE_VERSION }}
+          BASE_TAG_NAME: v${{ needs.bump-version.outputs.prev_version }}
+      - name: Create GitHub release
+        uses: softprops/action-gh-release@v1
+        with:
+          tag_name: v${{ env.PACKAGE_VERSION }}
+          files: elementor-*.zip
+          prerelease: ${{ github.event.client_payload.pre_release }}
+          body_path: temp-changelog.txt
+          repository: ${{ github.event.client_payload.repository}}
```

Comment:
Use a global variable, it can be from the action dispatch args

Reply:
done